include/triton/Dialect/TritonGPU/IR/TritonGPUOps.td

```diff
@@ -35,7 +35,11 @@ def TTG_AsyncWaitOp : TTG_Op<"async_wait"> {
 
   let extraClassDeclaration = [{
     static bool isSupported(int computeCapability) {
-      return computeCapability >= 80;
+      // ifdef USE_ROCM
+      return false;
+      // else
+      //return computeCapability >= 80;
```

Comment:
Why comment this line?

Reply:
agreed.

if those lines are uncommented and the '#' is added before the macros, this should work.